src/Stryker.Core/Stryker.Core/Options/Inputs/AzureFileStorageSasInput.cs

```diff
@@ -18,8 +20,16 @@ public string Validate(BaselineProvider baselineProvider)
                     throw new InputException("The azure file storage shared access signature is required when azure file storage is used for dashboard compare.");
                 }
 
-                // Normalize the SAS
-                return SuppliedInput.Replace("?sv=", "");
```

Comment:
This technically makes the change backwards incompatible, we will have to discuss if we want to make a breaking change at this point. I will get back to you on that.